Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -191,6 +194,38 @@ public sealed partial class HumanoidProfileEditor : Control
 
             #endregion Species
 
+            // Parkstation-HeightSlider Start
+            #region Height
+
+            var prototype = _speciesList.Find(x => x.ID == Profile?.Species) ?? _speciesList.First();
+
+            _heightSlider.MinValue = prototype.MinHeight;
+            _heightSlider.MaxValue = prototype.MaxHeight;
+            _heightSlider.Value = Profile?.Height ?? prototype.DefaultHeight;
+            CHeightLabel.Text = Loc.GetString("humanoid-profile-editor-height-label", ("height", _heightSlider.Value));
+
+            _heightSlider.OnValueChanged += args =>
+            {
+                if (Profile is null)
+                    return;
+
+                prototype = _speciesList.Find(x => x.ID == Profile.Species) ?? _speciesList.First(); // Just in case
+
+                var value = Math.Clamp(args.Value, prototype.MinHeight, prototype.MaxHeight);
+                var stringValue = Regex.Replace(value.ToString(CultureInfo.InvariantCulture), @"\.([0-9][0-9]).*", ".$1"); // Hide the extra decimals
```

Comment:
This seems overly complicated? What's it doin?

Reply:
Probably a 3am code moment, it's removing the insane floating point inaccuracy